--- CMakeLists_orig.txt	2016-02-02 23:08:44.000000000 +0100
+++ CMakeLists.txt	2017-06-28 18:50:45.349326700 +0200
@@ -812,7 +812,8 @@
 check_symbol_exists(setsockopt     "${CURL_INCLUDES}" HAVE_SETSOCKOPT)
 
 # symbol exists in win32, but function does not.
-check_function_exists(inet_pton HAVE_INET_PTON)
+# check_function_exists(inet_pton HAVE_INET_PTON)
+set(HAVE_INET_PTON 0)
 
 # sigaction and sigsetjmp are special. Use special mechanism for
 # detecting those, but only if previous attempt failed.
@@ -1197,3 +1198,4 @@
     file(APPEND "${CURL_SLN_FILENAME}" "\n# This should be regenerated!\n")
   endif()
 endif()
+
